screener-e2e/examples/test/specs/screener.simple.e2e.spec.js

```diff
@@ -0,0 +1,9 @@
+describe('My Login application', () => {
+    it('should take snapshot', () => {
+        browser.url('https://screener.io');
```

Comment:
Why not use the demo app, all examples in this repo are with the demo app.

Reply:
Our demo app isn't good for visual testing because it's too static and not realistic for the real world. See https://github.com/saucelabs/sample-app-web/issues/32